tests/Kernel/fixtures/rendering.yml

```diff
@@ -1672,7 +1754,7 @@
       "div.ecl-card__image": 0
       'h1.ecl-card__title': 1
       'div.ecl-card__description': 0
-      'footer.ecl-card__footer': 0
+      'footer.ecl-card__footer': 1
```

Comment:
We shouldn't change this test, if footer is not set then we shouldn't have it displayed. Maybe this was a regression in ecl twing, in which case we need to patch the related component in order to make sure that the original version of this test passes.